h5py/h5p.pyx

```diff
@@ -376,6 +376,14 @@ cdef class PropFCID(PropOCID):
         H5Pget_link_creation_order(self.id, &flags)
         return flags
 
+    @with_phil
+    def set_file_space_strategy(self, unsigned int strategy, bint persist, unsigned long long threshold):
```

Comment:
And this should be inside a version check, like this:

https://github.com/h5py/h5py/blob/84d7b7e58177e2c1b63ae37115bb4a88f6a90049/h5py/h5p.pyx#L1600-L1602

Reply:
PS it might be good to expose the corresponding get method while you're doing this.